src/services/config-service.ts

```
@@ -0,0 +1,180 @@
+export class ConfigService {
+  private static readonly CONFIG_DIR = '.function-indexer';
+  private static readonly CONFIG_FILE = 'config.json';
+  private static readonly INDEX_FILE = 'index.jsonl';
+  private static readonly DEFAULT_VERSION = '1.0.0';
+  
+  /**
+   * Get the configuration directory path
+   */
+  static getConfigDir(projectRoot?: string): string {
+    const root = projectRoot || process.cwd();
+    return path.join(root, this.CONFIG_DIR);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)